odbc_fdw.c

```diff
@@ -615,117 +602,109 @@ odbcGetTableSize(char *svr_dsn, char *svr_driver, char *svr_host, char *svr_port
                  char *svr_database, char *svr_schema, char *svr_table,
                  char *username, char *password, char *sql_count, unsigned int *size)
 {
-    SQLHENV env;
-    SQLHDBC dbc;
-    SQLHSTMT stmt;
-    SQLRETURN ret;
+	SQLHENV env;
+	SQLHDBC dbc;
+	SQLHSTMT stmt;
+	SQLRETURN ret;
 
-    StringInfoData	conn_str;
-    StringInfoData	sql_str;
-    SQLCHAR OutConnStr[1024];
-    SQLSMALLINT OutConnStrLen;
+	StringInfoData  conn_str;
+	StringInfoData  sql_str;
+	SQLCHAR OutConnStr[1024];
+	SQLSMALLINT OutConnStrLen;
 
-    SQLUBIGINT table_size;
-    SQLLEN indicator;
+	SQLUBIGINT table_size;
+	SQLLEN indicator;
 
-    StringInfoData name_qualifier_char;
-    StringInfoData quote_char;
+	StringInfoData name_qualifier_char;
+	StringInfoData quote_char;
 
-    const char* schema_name = svr_schema;
+	const char* schema_name = svr_schema;
 
-    if (!schema_name || !*schema_name)
-    {
-      schema_name = svr_database;
-    }
+	if (!schema_name || !*schema_name)
+	{
+	       schema_name = svr_database;
+	}
 
-    /* Construct connection string */
-    odbcConnStr(&conn_str, svr_dsn, svr_driver, svr_host, svr_port, svr_database, username, password);
-#ifdef DEBUG
-    elog(NOTICE, "Connection string: %s", conn_str.data);
```

Comment:
Yes, I removed most traces and left just the function names as they're entered.
This particular information could be interesting, but I thought I'll reintroduce traces as they're needed.